src/Config/RunConfig.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Project} from '../Project';
+import {BuilderCfgFile} from '../Project/BuilderCfgFile';
+import {Utils} from '../Utils';
+
+export function runConfig(payload: any): void {
```

Comment:
Q) whay is payload, can you change `any` to that type?

Reply:
The payload will be oneToolList. Which has whole tools.
I can define type and use it instead of any.